RELEASE-NOTES.md

```diff
@@ -10,144 +10,156 @@ Instead update the vNext section until 4.0.0 is out.
 ## PyMC vNext (4.0.0b1 → ... → 4.0.0b6 → 4.0.0)
 ⚠ The changes below are the delta between the releases `v3.11.5` →...→ `v4.0.0`.
 
-### Not-yet working features
-We plan to get these working again, but at this point their inner workings have not been refactored.
-- Timeseries distributions (see [#4642](https://github.com/pymc-devs/pymc/issues/4642))
-- Nested Mixture distributions (see [#5533](https://github.com/pymc-devs/pymc/issues/5533))
-- `pm.sample_posterior_predictive_w` (see [#4807](https://github.com/pymc-devs/pymc/issues/4807))
-- Partially observed Multivariate distributions (see [#5260](https://github.com/pymc-devs/pymc/issues/5260))
+### Do not miss 🚨
 
-Also check out the [milestones](https://github.com/pymc-devs/pymc/milestones) for a potentially more complete list.
+  - ⚠ The library is now named, installed and imported as "pymc". For example: `pip install pymc`.
+  - ⚠ Theano-PyMC has been replaced with Aesara, so all external references to `theano`, `tt`, and `pymc3.theanof` need to be replaced with `aesara`, `at`, and `pymc.aesaraf` (see [4471](https://github.com/pymc-devs/pymc/pull/4471)).
+  - ⚠ Random seeding behavior changed (see [#5787](https://github.com/pymc-devs/pymc/pull/5787))!
+    - Sampling results will differ from those of V3 when passing the same `random_seed` as before. They will be consistent across subsequent V4 releases unless mentioned otherwise.
+    - Sampling functions no longer respect user-specified global seeding! Always pass `random_seed` to ensure reproducible behavior.
+  - ⚠ BART was removed from the main package [#5566](https://github.com/pymc-devs/pymc/pull/5566). It is now available from [pymc-experimental](https://github.com/pymc-devs/pymc-experimental).
+  - ⚠ The GLM submodule was removed, please use [Bambi](https://bambinos.github.io/bambi/) instead.
+  - ⚠ PyMC now requires Scipy version `>= 1.4.1` (see [4857](https://github.com/pymc-devs/pymc/pull/4857)).
+
+  #### v3 features not yet working in v4 ⏳
+
+  We plan to get these working again, but at this point their inner workings have not been refactored.
+  - Timeseries distributions (see [#4642](https://github.com/pymc-devs/pymc/issues/4642))
```

Comment:
Mention them by name: MvNormalRandomWalk, MvStudentTRandomWalk, GARCH11 and weird name one xD

Reply:
Ha ha but that's the point: what's the name of the last one?? 😆